appveyor.yml

```diff
@@ -1,60 +1,65 @@
-image: Visual Studio 2019
+# Source: https://github.com/krlmlr/r-appveyor/blob/master/appveyor.yml
 
-version: '{build}'
+# DO NOT CHANGE the "init" and "install" sections below
 
 # Download script file from GitHub
 init:
-  - ps: Update-AppveyorBuild -Version "$($env:ospsuite_version).$($env:appveyor_build_version)"
-  - ps: |
-      $ErrorActionPreference = "Stop"
-      Invoke-WebRequest http://raw.github.com/krlmlr/r-appveyor/master/scripts/appveyor-tool.ps1 -OutFile "..\appveyor-tool.ps1"
-      Import-Module '..\appveyor-tool.ps1'
+  ps: |
+        Get-Date
+        $ErrorActionPreference = "Stop"
 
 install:
-  - ps: Bootstrap
-  - git submodule update --init --recursive
+  ps: |
+        Import-Module '.\scripts\appveyor-tool.ps1'
+        Bootstrap
 
 environment:
   USE_RTOOLS: true
-  NOT_CRAN: true
-  KEEP_VIGNETTES: true
+  R_VERSION: "4.1.0"
   R_ARCH: x64
-  R_VERSION: "4.0.5"
-  COVERALLS_TOKEN:
-    secure: xIz/WZT0ex3bs/CMBJTzzdXLhl3sqfSqJ3MshlSY03pZKuyYQN7Z1FprVgnlFMUZ
-
+  KEEP_VIGNETTES: true
+  NOT_CRAN: true
 
+version: "$(app_version).{build}"
```

Comment:
ok this is good. Where is app_version defined?

Reply:
Forgot to add it. Fixed it now.